root/meta/MakeProject/CMakeLists.txt

```diff
@@ -20,24 +20,17 @@ ROOTTEST_ADD_TEST(runaliceesd
 
 ROOTTEST_GENERATE_DICTIONARY(stl_makeproject_test stl_makeproject_test.h LINKDEF stl_makeproject_test_linkdef.h)
 
-if(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/module.modulemap")
-  # For C++ modules builds, module.modulemap is generated during configuration time.
+if(ROOT_runtime_cxxmodules_FOUND)
+  # For C++ modules builds, module.modulemap is generated during configuration time
+  # and is not needed during execution of this test, but later in other tests it needs to be back.
   ROOTTEST_ADD_TEST(examples
-                   COPY_TO_BUILDDIR stl_makeproject_test.h
+                   COPY_TO_BUILDDIR stl_makeproject_test.h disable_modulemap.sh enable_modulemap.sh
+                   PRECMD ./disable_modulemap.sh ${CMAKE_CURRENT_BINARY_DIR}
                    MACRO create_makeproject_examples.C
-                   POSTCMD ${CMAKE_COMMAND} -E rename ${CMAKE_CURRENT_BINARY_DIR}/module.modulemap ${CMAKE_CURRENT_BINARY_DIR}/disabled.module.modulemap
+                   POSTCMD ./enable_modulemap.sh ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Does this work on Windows?

Reply:
Modules are only working on Unix currently.

Reply:
but that is now yet-another-hurdle in supporting module on windows ....